libafl/src/state/mod.rs

```diff
@@ -556,3 +565,24 @@ where
         unimplemented!()
     }
 }
+
+impl<C, FT, I, R, SC> HasMOpt for StdState<C, FT, I, R, SC>
```

Comment:
@tokatoka can you remove the HasMOpt impl? It is not mandatory to use MOpt, so StdState will not have a mopt metadata every times. In your code, just use metadata().get